packages/@aws-cdk/aws-msk-alpha/lib/cluster.ts

```diff
@@ -103,6 +103,12 @@ export interface ClusterProps {
    * @default - 1000 GiB EBS volume
    */
   readonly ebsStorageInfo?: EbsStorageInfo;
+  /**
+   * Local or Tiered storage configuration for the brokers
+   *
+   * @default - undefined
+   */
```

Comment:
Thanks for raising this so that I can better understand what the desired effect is.

I am no expert, but currently the CloudFormation templates show no indication of the `storageMode`, unless storage mode has been set to `TIERED`. I have noticed this while deploying numerous MSK clusters via CDK over the past month. This "vacancy" of the `storageMode` was something I thought should continue, which is why I was trying to leave the default as `none` or `undefined`, rather than changing the default behavior.

> Adding a cluster from the Console selects LOCAL as the default option so I think CloudFormation will resolve to that if not defined.

`aws-cli` describe-cluster always shows the storageMode of a cluster. As you say, the default is `LOCAL`. Is it best that I explicitely define this now as the default, as you have suggested?

Reply:
Thanks for the follow-up.
Setting `@default - StorageMode.LOCAL` in this case is just an indication of the default behavior for the deployed app (we don't need to manage it via CDK).
The CDK-produced CloudFormation won't contain the `storageMode`, but the deployed cluster will be `LOCAL`.